python/lsst/analysis/tools/analysisPlots/skySource.py

```diff
@@ -65,16 +65,28 @@ def setDefaults(self):
         self.prep.selectors.skySourceSelector = SkySourceSelector()
 
         self.process.buildActions.hist_psf_flux = LoadVector(vectorKey="psfFlux")
-        self.process.buildActions.hist_09_flux = LoadVector(vectorKey="ap09Flux")
+        self.process.buildActions.hist_ap09_flux = LoadVector(vectorKey="ap09Flux")
+        self.process.buildActions.hist_gaap1p0_flux = LoadVector(vectorKey="gaap1p0Flux")
         self.process.buildActions.hist_psf_sn = SNCalculator(fluxType="psfFlux")
-        self.process.buildActions.hist_09_sn = SNCalculator(fluxType="ap09Flux")
+        self.process.buildActions.hist_ap09_sn = SNCalculator(fluxType="ap09Flux")
+        self.process.buildActions.hist_gaap1p0_sn = SNCalculator(vectorKey="gaap1p0Flux")
 
         self.produce = HistPlot()
 
         self.produce.panels["panel_flux"] = HistPanel()
         self.produce.panels["panel_flux"].label = "Flux (nJy)"
-        self.produce.panels["panel_flux"].hists = dict(hist_psf_flux="psfFlux", hist_09_flux="ap09Flux")
+        self.produce.panels["panel_flux"].hists = dict(
+            hist_psf_flux="psfFlux", hist_ap09_flux="ap09Flux", hist_gaap1p0_flux="gaap1p0Flux"
+        )
+
```

Comment:
I don't think you want this newline here do you, if you want to keep all of the `"panel_flux"` produce code block together (as below with the `"panel_sn"` produce code block).